index.bs

```diff
@@ -34,8 +34,19 @@ urlPrefix: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure; spec
             text: boolean; url: #section-3.3.4
             text: string; url: #section-3.3.3
             text: list; url: #section-3.1
+            text: serializing a list; url: #ser-list
```

Comment:
I don't see `#ser-list` anywhere in the draft, should we link to `#section-4.1.1` instead?

(https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-19#section-4.1.1)
```suggestion
            text: serializing a list; url: #section-4.1.1
```